src/NuGet.Core/NuGet.CommandLine.XPlat/Utility/ProjectPackagesPrintUtility.cs

```diff
@@ -211,5 +211,13 @@ internal static string[] BuildTableHeaders(bool printingTransitive, bool outdate
 
             return result.ToArray();
         }
+
+        internal static void PrintSources(IEnumerable<PackageSource> packageSources)
+        {
+            foreach (var source in packageSources)
+            {
+                Console.WriteLine("   " + source.Source);
```

Comment:
just fyi, you don't need to change it.
$"    {source.Source}" - string interpolation is usually more efficient than string concatenation.